### PR TITLE
Importing en.inc.php from Legacy OC

### PR DESCRIPTION
**Pull request by phillf**

### CODE REVIEW

**Review by visuallyadvised — APPROVED** [review]

**Comment by kevingorman1000** [comment]
There goes my 0 code smells score :'( Guess more things for me to sort hahaha

File: core/locale/en.inc.php [hunk 1]
```diff
@@ -0,0 +1,570 @@
+<?php
+$arrLang = array();
+
+$arrLang['alert_admin_email_wrong'] = "Admin email has wrong format! Please re-enter.";
+$arrLang['alert_min_version_db'] = "This program requires at least version _DB_VERSION_ of _DB_ installed (current version is _DB_CURR_VERSION_). You cannot proceed the installation.";
+$arrLang['alert_min_version_php'] = "This program requires at least version _PHP_VERSION_ of PHP installed (current version is _PHP_CURR_VERSION_). You cannot proceed the installation.";
```
Comment (Cambridgeport90) [inline core/locale/en.inc.php:6]: Something tells me this should say "You cannot proceed with the installation". If that's just the English grad in me needing to shut up, also let me know.

Comment (Cambridgeport90) [inline core/locale/en.inc.php:6]: same thing.

File: core/locale/en.inc.php [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+$arrLang['error_pdo_support'] = "This installation requires <b>PDO extension</b> installed.";
+$arrLang['error_sql_executing'] = "SQL execution error! Please Turn debug mode On and check carefully a syntax of your SQL dump file.";
+$arrLang['error_server_requirements'] = "This installation requires _SETTINGS_NAME_ settings turned on/installed.";
+$arrLang['error_vd_support'] = "This installation requires Virtual Directory support turned ON.";
+
+$arrLang['admin_access_data'] = "Administrator Account Info";
+$arrLang['admin_access_data_descr'] = "(you need this to enter the protected admin area)";
+$arrLang['admin_email'] = "Email";
+$arrLang['admin_email_info'] = "The administrator email which will be used to login to this accout.
+
```
Comment (Cambridgeport90) [inline core/locale/en.inc.php:36]: Think that should be account.

**Review by Cambridgeport90 — COMMENTED** [review]
I have to say, the extra information in this file shows that we have a leg up when it comes to providing guidance during installation; as long as folks read, there's no way that they won't understand what's happening. This is better than some professional web installers I have seen. Other than the typos I flagged, we're good.

**Pull request merged**